plugins/mobile-plugin/src/main/java/ru/sbtqa/tag/pagefactory/mobile/utils/SwipeUtils.java

```diff
@@ -147,14 +153,14 @@ public static void swipeToText(DirectionStrategy direction, String text, MatchSt
      */
     public static void swipeToText(DirectionStrategy direction, String text, MatchStrategy strategy, int depth) throws SwipeException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 20 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)